python/src/opendp/mod.py

```diff
@@ -111,6 +111,10 @@ def __rshift__(self, other: Union["Function", "Transformation"]) -> "Measurement
         if isinstance(other, Transformation):
             other = other.function
 
+        if not isinstance(other, Function):
+            from opendp.core import new_function
+            other = new_function(other, TO="ExtrinsicObject")
+
         if isinstance(other, Function):
```

Comment:
It makes it so that you can do: `my_meas >> (lambda arg: arg * 2)`. The postprocessor doesn't have to be wrapped in `dp.new_function(lambda arg: arg * 2, TO="ExtrinsicObject")`

Reply:
I think that explains the `if not isinstance(other, Function)` above but I'm curious about `if isinstance(other, Function)` -- It seems like it would always pass now? Can't use GH suggestion because of scope, but would this work?

```diff
-        if isinstance(other, Function):
-            from opendp.combinators import make_chain_pm
-            return make_chain_pm(other, self)
-
-        raise ValueError(f"rshift expected a postprocessing transformation, got {other}")
+        from opendp.combinators import make_chain_pm
+        return make_chain_pm(other, self)
```